packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.ts

```diff
@@ -189,7 +189,9 @@ export class CryptoWatchPriceFeed extends PriceFeedInterface {
     // why we prefer `before` periods to `after` ones. Similar reasoning is why we default to a period's open price to
     // its close price. We prefer prices that occurred before the target time.
     else {
-      returnPrice = this.invertPrice ? this._invertPriceSafely(match.openPrice) : match.openPrice;
+      // Use period close price when request time matches close time. In all other cases use open price instead.
+      const matchedPrice = time == match.closeTime ? match.closePrice : match.openPrice;
```

Comment:
I acknowledge not using strict equality operator as `time` is number, but `closeTime` is string. Should I have converted `closeTime` to number instead?

Reply:
I would convert them to the same type and use strict equality for completeness

Reply:
or just do time.toString()

Reply:
My assumptions were wrong and this was a problem only when running `getHistoricalPrice.js` and passing `--time` on comandline that. Next commit fixes that.